### DISCUSSION TITLE
Working on Todo Module

### DISCUSSION THREAD

**Discussion by PizzaConsole** [oqtane/oqtane.framework#761] [Show and tell]
Hey all, I am working on a Module for a Todo List. I am trying out a variety of different thing in this project to push Oqtane to the test. I want to be able to use this repo as a template for my future Modules as well so its easy to startup a new project!

Come check it out if you'd like. I would LOVE feedback! 

[PoisnFang.Todo](https://github.com/PoisnFang/PoisnFang.Todo)

Some of the things I have implemented (or will try to implement) are:

- Generic Repository
- Unit of work
- EF Core Migrations
- Dynamic routing inside the module using the url parameters
- Initializing the database on module install from C# rather than SQL script
- Custom permissions per module

**Comment by sbwalker** [comment]
very cool!
